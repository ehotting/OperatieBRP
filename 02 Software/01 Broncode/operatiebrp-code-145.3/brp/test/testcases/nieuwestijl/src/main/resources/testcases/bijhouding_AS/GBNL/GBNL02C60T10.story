Meta:
@status                 Klaar
@usecase                UCS-BY.0.AS

Narrative: R1746 Leeftijdsverschil kinderen van zelfde OUWKIG moet minstens 9 maanden zijn

Scenario:   DatumGeboorte van een ander Kind van diezelfde Ouder verschilt 305 dagen
            LT: GBNL02C60T10

Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL-AS/LO3PL-GBNL/GBNL02C60-001.xls
Given enkel initiele vulling uit bestand /LO3PL-AS/LO3PL-GBNL/GBNL02C60-002.xls

When voer een bijhouding uit GBNL02C60T10a.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Geslaagd

When voer een bijhouding uit GBNL02C60T10b.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Geslaagd
Then is het antwoordbericht gelijk aan /testcases/bijhouding_AS/GBNL/expected/GBNL02C60T10.xml voor expressie //brp:bhg_afsRegistreerGeboorte_R