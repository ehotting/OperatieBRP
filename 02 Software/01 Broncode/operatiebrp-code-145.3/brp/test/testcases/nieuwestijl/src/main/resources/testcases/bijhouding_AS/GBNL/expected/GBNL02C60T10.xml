<brp:bhg_afsRegistreerGeboorte_R xmlns:brp="http://www.bzk.nl/brp/brp0200">
    <brp:stuurgegevens>
        <brp:zendendePartij>199901</brp:zendendePartij>
        <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
        <brp:referentienummer>*</brp:referentienummer>
        <brp:crossReferentienummer>00000000-0000-0000-0042-200000000104</brp:crossReferentienummer>
        <brp:tijdstipVerzending>*</brp:tijdstipVerzending>
    </brp:stuurgegevens>
    <brp:resultaat>
        <brp:verwerking>Geslaagd</brp:verwerking>
        <brp:bijhouding>Verwerkt</brp:bijhouding>
        <brp:hoogsteMeldingsniveau>Waarschuwing</brp:hoogsteMeldingsniveau>
    </brp:resultaat>
    <brp:meldingen>
        <brp:melding brp:objecttype="Melding" brp:referentieID="CI_relatie_1">
            <brp:regelCode>R1746</brp:regelCode>
            <brp:soortNaam>Waarschuwing</brp:soortNaam>
            <brp:melding>*</brp:melding>
        </brp:melding>
    </brp:meldingen>
    <brp:geboorteInNederland brp:objecttype="AdministratieveHandeling">*</brp:geboorteInNederland>
</brp:bhg_afsRegistreerGeboorte_R>
